api/censuses.go

```diff
@@ -529,18 +578,31 @@ func (a *API) censusProofHandler(msg *apirest.APIdata, ctx *httprouter.HTTPConte
 	if err != nil {
 		return err
 	}
-	keyHash, err := ref.Tree().Hash(key)
+
+	leafKey := key
+	if ref.CensusType != int32(models.Census_ARBO_POSEIDON) {
```

Comment:
Please document why we are not performing a hash if the census type is ARBO_POSEIDON

Reply:
Included!